### DISCUSSION TITLE
🧩 FEATURE REQUEST:  daily.edu

### DISCUSSION THREAD

**Discussion by WhyIsEmerald** [dailydotdev/daily#1492] [Feature Requests]
### Concise Description

daily.dev but for science and other educational news

### Description

This is a bit off topic but I would love to have a science version of daily.dev and so would my peers so please make a section for science of another extension called daily.edu or something.

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by idoshamun** [comment]
Hi @emerald-developer,

Thanks for the suggestion. You can use the [#science](https://app.daily.dev/tags/science) tag page or create a custom feed with science-related tags. We have no plans to create a dedicated science version of daily.dev
